src/database_metrics/operations.py

```diff
@@ -0,0 +1,75 @@
+"""Operations for which metrics are captured for."""
```

Comment:
This edit may not yield a good docstring, but it's less bad:

```suggestion
"""We can collect metrics on the data transformation procedures defined in this file."""
```

* this repo is named "operations" and it's a different "operations" than the "operations" meant here. Avoid overloading if possible
* for which X are Y for? no

Reply:
Good call, will rename. `actions.py`?

Reply:
sgtm

Reply:
changed